src/app/components/ProposalInfo/index.tsx

```diff
@@ -54,54 +55,81 @@ const ContentText = styled(Typography)`
   font-size: 16px;
 `;
 
-export default function ProposalInfo({ proposal }: { proposal: ProposalInterface }) {
-  const {
-    name: title,
-    description,
-    startDay,
-    endDay,
-    status,
-    for: approvePercentage,
-    against: rejectPercentage,
-    uniqueApproveVoters,
-    uniqueRejectVoters,
-  } = proposal;
+export const StyledSkeleton = styled(Skeleton)`
+  background-color: rgba(44, 169, 183, 0.2) !important;
+`;
+
+export default function ProposalInfo({ proposal }: { proposal: ProposalInterface | null }) {
+  const title = proposal?.name;
+  const description = proposal?.description;
+  const startDay = proposal?.startDay;
+  const endDay = proposal?.endDay;
```

Comment:
why did you refactor the code?

Reply:
I couldn't properly destructure the props after allowing them to be null